From c88d340145b63c1dfb498836a0c5f0bcd8d6e60c Mon Sep 17 00:00:00 2001
From: "W.C.A. Wijngaards" <wouter@nlnetlabs.nl>
Date: Fri, 23 Jul 2021 09:59:21 +0200
Subject: [PATCH] - Fix free on shutdown of XoT SSL context.

diff --git a/xfrd.c b/xfrd.c
index 6f5768f7..b4d99c00 100644
--- a/xfrd.c
+++ b/xfrd.c
@@ -402,6 +402,8 @@ xfrd_shutdown()
 	daemon_remote_delete(xfrd->nsd->rc); /* ssl-delete secret keys */
 	if (xfrd->nsd->tls_ctx)
 		SSL_CTX_free(xfrd->nsd->tls_ctx);
+	if (xfrd->tcp_set->ssl_ctx)
+		SSL_CTX_free(xfrd->tcp_set->ssl_ctx);
 #endif
 #ifdef USE_DNSTAP
 	dt_collector_close(nsd.dt_collector, &nsd);
