diff -uNr nsd-2.2.0.orig/Makefile.in nsd-2.2.0/Makefile.in
--- nsd-2.2.0.orig/Makefile.in	2005-01-06 13:22:07.000000000 +0100
+++ nsd-2.2.0/Makefile.in	2005-01-18 15:55:45.863723040 +0100
@@ -134,22 +134,22 @@
 	$(EDIT) nsdc.conf.sample.in > nsdc.conf.sample
 
 install: all
-	$(INSTALL) -d $(sbindir)
-	$(INSTALL) -d $(configdir)
-	$(INSTALL) -d $(mandir)
-	$(INSTALL) -d $(mandir)/man8
-	$(INSTALL) nsd $(sbindir)/nsd
-	$(INSTALL) zonec $(sbindir)/zonec
-	$(INSTALL) nsdc.sh $(sbindir)/nsdc
-	$(INSTALL) nsd-notify $(sbindir)/nsd-notify
-	$(INSTALL) nsd-xfer $(sbindir)/nsd-xfer
-	$(INSTALL_DATA) nsd.8 $(mandir)/man8
-	$(INSTALL_DATA) nsdc.8 $(mandir)/man8
-	$(INSTALL_DATA) zonec.8 $(mandir)/man8
-	$(INSTALL_DATA) nsd-notify.8 $(mandir)/man8/nsd-notify.8
-	$(INSTALL_DATA) nsd-xfer.8 $(mandir)/man8/nsd-xfer.8
-	$(INSTALL_DATA) nsdc.conf.sample $(configfile).sample
-	$(INSTALL_DATA) nsd.zones.sample $(zonesfile).sample
+	$(INSTALL) -d $(DESTDIR)/$(sbindir)
+	$(INSTALL) -d $(DESTDIR)/$(configdir)
+	$(INSTALL) -d $(DESTDIR)/$(mandir)
+	$(INSTALL) -d $(DESTDIR)/$(mandir)/man8
+	$(INSTALL) nsd $(DESTDIR)/$(sbindir)/nsd
+	$(INSTALL) zonec $(DESTDIR)/$(sbindir)/zonec
+	$(INSTALL) nsdc.sh $(DESTDIR)/$(sbindir)/nsdc
+	$(INSTALL) nsd-notify $(DESTDIR)/$(sbindir)/nsd-notify
+	$(INSTALL) nsd-xfer $(DESTDIR)/$(sbindir)/nsd-xfer
+	$(INSTALL_DATA) nsd.8 $(DESTDIR)/$(mandir)/man8
+	$(INSTALL_DATA) nsdc.8 $(DESTDIR)/$(mandir)/man8
+	$(INSTALL_DATA) zonec.8 $(DESTDIR)/$(mandir)/man8
+	$(INSTALL_DATA) nsd-notify.8 $(DESTDIR)/$(mandir)/man8/nsd-notify.8
+	$(INSTALL_DATA) nsd-xfer.8 $(DESTDIR)/$(mandir)/man8/nsd-xfer.8
+	$(INSTALL_DATA) nsdc.conf.sample $(DESTDIR)/$(configfile).sample
+	$(INSTALL_DATA) nsd.zones.sample $(DESTDIR)/$(zonesfile).sample
 
 uninstall:
 	@echo
